docs/data/core-edit-post.md

```diff
@@ -0,0 +1,271 @@
+# **core/edit-post**: The Editor's UI Data
+
+## Selectors 
+
+### getEditorMode
+
+Returns the current editing mode.
+
+*Parameters*
+
+ * state: Global application state.
```

Comment:
This is a different description to the previous one used for all the states. Is that because the other one was the `core/data` state but this isn't? I'm confused. It might help to link to the state it expects for each function.